amrit-local-setup/README.md

```diff
@@ -0,0 +1,68 @@
+# AMRIT Local Environment Setup Guide
+
+## System Requirements
+
+### Mandatory Dependencies
+
+- Docker Engine + Docker Compose
+- Maven 3.6+
+- Git version control
+- OpenJDK 17+
+
+## Architecture Overview
+
+The setup leverages containerization for consistent development environments across the team. Core services are orchestrated via Docker, with MySQL and Redis instances running in isolated containers.
+
+## Deployment Steps
+
+### 1. Container Orchestration
+
+Initialize the containerized services:
+
+```bash
+docker-compose up --build
+```
+
+#### Service Endpoints
+
+- MySQL Instance: `localhost:3307`
+- Redis Instance: `localhost:4407`
```

Comment:
Yeah still it is inside the container but to the host port it is mapped to a different port so that host redis port will not conflict with contarised redis port!